cmd/stellar-rpc/internal/daemon/daemon.go

```diff
@@ -292,6 +293,19 @@ func createIngestService(cfg *config.Config, logger *supportlog.Entry, daemon *D
 		logger.WithError(err).Error("could not run ingestion. Retrying")
 	}
 
+	var backend ledgerbackend.LedgerBackend = daemon.core
+	if cfg.LoadTestFile != "" {
+		daemon.Logger().
+			WithField("path", cfg.LoadTestFile).
+			Warnf("Ingestion will run with load testing")
+
+		backend = loadtest.NewLedgerBackend(loadtest.LedgerBackendConfig{
+			NetworkPassphrase:   cfg.NetworkPassphrase,
+			LedgersFilePath:     cfg.LoadTestFile,
+			LedgerCloseDuration: time.Second * 5000,
```

Comment:
in horizon we have 2 more configuration parameters:

https://github.com/stellar/go/blob/master/services/horizon/cmd/ingest.go#L88-L112

- `merge` - configures whether we merge real ledgers with the synthetic ledgers in the load test
- `close-duration` - configures the close time. This is pretty important because we can use it to test if we're able to handle a faster close duration. We configure this value to 2 seconds for the horizon load tests

Reply:
Done! Added both flags and am targeting a feature branch now. Restoration will take more effort and I'll do it in a separate PR :+1: